src/main/java/org/nest/units/unitrepresentation/SIData.java

```diff
@@ -61,7 +61,7 @@
   private static String[] SIUnitsDerivedRaw =
       {"Hz","N","Pa","J","W","C","V","F","Ohm","S","Wb","T","H","lm","lx","Bq","Gy","Sv","kat"};
   private static String[] SIPrefixesRaw =
-      {"da","h","k","M","G","T","P","E","Z","Y","d","c","m","mu","n","p","f","a","z","y"};
+      {"k","M","G","T","P","E","Z","Y","m","mu","n","p","f","a","z","y"};
```

Comment:
I think the removal of these units is not a problem in usual neuroscience applications.

Regarding `mu`: already extended ASCII had `µ` at position 230, so I see no principle problem with using it. If not, we should rather use `u` instead of `mu` to at least make all prefixes consistently one-letter.